common_code/solver_cg_s_step.h

```diff
@@ -87,19 +89,21 @@ class SolverCGSStep
 
     unsigned int c = 1;
 
+    times[0] += time.wall_time();
```

Comment:
I am fine to close this PR. A question would be if we want to measure the time for initialization?